src/main/java/page/AbstractPage.java

```diff
@@ -0,0 +1,22 @@
+package page;
+
+import org.openqa.selenium.WebDriver;
+
+public abstract class AbstractPage {
+
+    protected WebDriver driver;
+//    protected WebDriverWait wait;
+//    JavascriptExecutor js;
+
+    protected abstract AbstractPage openPage();
+    protected final int WAIT_TIMEOUT_SECONDS = 10;
+
+    protected AbstractPage(WebDriver driver){
+        this.driver = driver;
+//        wait = new WebDriverWait(driver, 15, 50);
```

Comment:
It was a good idea to move repeating logic to the super class. Uncomment it and use it instead of repeat the same steps in child classes.